options.go

```diff
@@ -334,3 +334,69 @@ func WithWaitStrategyAndDeadline(deadline time.Duration, strategies ...wait.Stra
 		return nil
 	}
 }
+
```

Comment:
In case there is a discussion about variadic Vs single arguments in the new options, I left a comment here: https://github.com/testcontainers/testcontainers-go/pull/3071#discussion_r2022567464